.github/workflows/tests.yml

```diff
@@ -11,6 +11,10 @@ on:
         description: "Fluence environment to run tests against"
         type: string
         default: "local"
+      user-fluence-dir-path:
+        description: "Path to the user's .fluence directory"
+        type: string
+        default: "/opt/actions-runner/_work/fluence-cli/fluence-cli/tmp/.fluence"
```

Comment:
Do you really need to specify it as an input to a workflow? I suggest doing it just with env variable, something like:
```
env:
  FORCE_COLOR: true
  RUST_PEER_IMAGE: "${{ inputs.rust-peer-image }}"
  FLUENCE_DIR: "${{ github.workspace }}/tmp/fluence"

Reply:
agree. No reason to specify this as input to workflow